foundation/okfplugins/grid_columns/templates/grid_columns_plugin.html

```diff
@@ -1,10 +1,14 @@
 {% load cms_tags %}
 <div class="sect -tighten-1 sect-generic mb-20">
     <div class="sect__content">
-      <div class="grid {{ grid_cols_class }} gap-10 md:gap-3">
-      {% for plugin in instance.child_plugin_instances %}
-          {% render_plugin plugin %}
-      {% endfor %}
+      <div class="grid md:grid-cols-{{ instance.columns }} gap-10 md:gap-3">
```

Comment:
Tailwind does not get along well with Dynamic Classes (https://tailwindcss.com/docs/content-configuration#dynamic-class-names) so we should avoid using interpolation.

That's why I created the `grid_cols_class` variable in the view.